lib/assert.js

```diff
@@ -554,6 +559,7 @@ function partiallyCompareArrays(actual, expected, comparedObjects) {
         } else {
           expectedCounts.set(zeroKey, count - 1);
         }
+        continue;
```

Comment:
The `continue` won't be needed, since the if statement will end and all other operations in the loop are also done. It won't impact the performance anymore and it is normally easier to read without `continue` statements.

```suggestion
```